contrib/completion/fish/docker.fish

```diff
@@ -26,12 +26,14 @@ end
 function __fish_print_docker_containers --description 'Print a list of docker containers' -a select
     switch $select
```

Comment:
This looks great; just one nitpick: there should be a `set -l filter` here to declare `filter` as local variable, otherwise we might use a global variable with the same name.